docker-test-env/reverse_proxy/README.md

```diff
@@ -5,9 +5,9 @@ The setups is as follows:
 `Selenium -> Browser -> Nginx Proxy -> dfx (local replica)`
 
 Nginx is used to terminate tls. This allows the use of the actual domain `https://identity.ic0.app` in the tests (dfx does not support `https`). Note that we cannot use `http://identity.ic0.app` because of preloaded HSTS.
-We still need the `--ignore-certificate-errors` because we don't add the certificates to the browser truststore.
+We still need the `--ignore-certificate-errors` because we don't add the certificates to the browser truststore. Having this option enabled also allows us to use the same certificate for all domains, even if the CN does not match.
 
-The certificates are self-signed and created using the following commands:
+The certificate is self-signed and created using the following commands:
```

Comment:
I'm assuming the certificate still needs to be well-formed? Does it need to be valid for any of the domains?

Reply:
Yes, otherwise nginx crashes on startup.

Reply:
> Does it need to be valid for any of the domains?
No

Reply:
Ack. Then I guess we can remove the openssl instructions and just copy/paste the dummy cert? 😅

Reply:
I would leave the instructions there in case somebody needs a new dummy cert.
E.g. I don't know how nginx will react to the dummy cert expiring.